uaclient/files/notices.py

```diff
@@ -0,0 +1,244 @@
+import os
+from collections import namedtuple
+from enum import Enum
+from typing import Any, Dict, List, Optional  # noqa: F401
+
+from uaclient import defaults, event_logger, messages, system
+from uaclient.files import UAFile
+
+event = event_logger.get_event_logger()
+NoticeFileDetails = namedtuple(
+    "NoticeFileDetails", ["order_id", "label", "is_permanent", "message"]
+)
+
+
+class NoticesDetails(NoticeFileDetails, Enum):
```

Comment:
minor bikeshed suggestion. I think the code reads slightly better with this rename
```suggestion
class Notice(NoticeFileDetails, Enum):
```